ci/ray_ci/automation/weekly_green_metric.py

```diff
@@ -25,7 +25,7 @@ def main(production: bool) -> None:
     init_global_config(bazel_runfile("release/ray_release/configs/oss_config.yaml"))
     blockers = TestStateMachine.get_release_blockers()
     logger.info(f"Found {blockers.totalCount} release blockers")
-
+    print(blockers.totalCount)
```

Comment:
rather than doing this, can we add an flag/arg `--check` and perform the check inside the script? rather than doing another redirection via bash?

Reply:
done!

Reply:
Ran a build on another identical branch (so I could add the branch queue in to test): https://buildkite.com/ray-project/release-automation/builds/172#_
It correctly asserts the release blocker count and fails the step.